readthedocs/storage/rclone.py

```diff
@@ -181,10 +182,12 @@ def __init__(
         self.env_vars = {
             "RCLONE_S3_PROVIDER": provider,
             "RCLONE_S3_ACCESS_KEY_ID": access_key_id,
-            "RCLONE_S3_SECRET_ACCESS_KEY": secret_acces_key,
+            "RCLONE_S3_SECRET_ACCESS_KEY": secret_access_key,
             "RCLONE_S3_REGION": region,
             "RCLONE_S3_LOCATION_CONSTRAINT": region,
         }
+        if session_token:
+            self.env_vars["RCLONE_S3_SESSION_TOKEN"] = session_token
```

Comment:
This could probably also use a comment linking to the dev docs about this.

Reply:
```suggestion
            # Pass the temporary sesion token to rclone via an environment variable
            # https://rclone.org/s3/#authentication
            # https://rclone.org/s3/#s3-session-token
            self.env_vars["RCLONE_S3_SESSION_TOKEN"] = session_token
```